# built in errors exercise

g = open('rui.txt')